src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -12,6 +12,20 @@
 
 {% block scripts %}
     <script src="{{ asset('bundles/packagistweb/js/view.js')}}"></script>
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/styles/monokai_sublime.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/highlight.min.js"></script>
+    <script>
+      $(document).ready(function() {
+        hljs.configure({
+          tabReplace: 4,
+          languages: ['apache', 'bash', 'ini', 'javascript', 'json', 'nginx', 'php', 'sql', 'css', 'html']
```

Comment:
Well, in most cases, packages are giving an explicit language in their markdown, because it is necessary to have highlighting on github (they don't have auto-detection).
It is too bad that the github markdown renderer does not include the needed class in the rendered HTML to make this info available. This would avoid the need for auto-detection